src/Misc/interfaces/iPerfViewer.ts

```diff
@@ -13,6 +16,7 @@ export interface IPerfPoint {
     value: number;
 }
 
+// TODO: REMOVE ONCE Everything is connected!

```

Comment:
To minimize the size of the PR I have marked these for removal and they will be removed in a later PR (When the canvasGraphService changes are in PR I will remove these interfaces).